.travis.yml

```diff
@@ -28,6 +28,16 @@ matrix:
 notifications:
   email: false
 
+before_install:
+  - export CXX="g++-6" CC="gcc-6"
```

Comment:
All code we write must also work with gcc5.

Reply:
Technically gcc4.8, but we're dropping that after pytorch 1.3

Reply:
fixed in the latest commit.